website/pages/style.css

```diff
@@ -15,23 +15,21 @@
 @import '~@hashicorp/react-button/dist/style.css';
 @import '~@hashicorp/react-call-to-action/dist/style.css';
 @import '~@hashicorp/react-case-study-slider/dist/style.css';
-@import '~@hashicorp/react-code-block/dist/style.css';
+@import '~@hashicorp/react-code-block/style.css';
 @import '~@hashicorp/react-consent-manager/dist/style.css';
 @import '~@hashicorp/react-content/dist/style.css';
 @import '~@hashicorp/react-docs-page/style.css';
 @import '~@hashicorp/react-docs-sidenav/dist/style.css';
-@import '~@hashicorp/react-enterprise-alert/dist/style.css';
+@import '~@hashicorp/react-enterprise-alert/style.css';
 @import '~@hashicorp/react-featured-slider/dist/style.css';
 @import '~@hashicorp/react-logo-grid/dist/style.css';
 @import '~@hashicorp/react-product-downloader/dist/style.css';
 @import '~@hashicorp/react-product-features-list/dist/style.css';
 @import '~@hashicorp/react-search/dist/style.css';
 @import '~@hashicorp/react-section-header/dist/style.css';
 @import '~@hashicorp/react-subnav/dist/style.css';
-@import '~@hashicorp/react-tabs/dist/style.css';
-@import '~@hashicorp/react-text-and-content/dist/style.css';
-@import '~@hashicorp/react-text-split-with-code/dist/style.css';
-@import '~@hashicorp/react-text-split/dist/style.css';
+@import '~@hashicorp/react-tabs/style.css';
+@import '~@hashicorp/react-text-split/style.css';
```

Comment:
`text-split-with-code` no longer ships any styles - they're all contained in the composed `text-split` and `code-block` components.

`text-and-content` has been deprecated and removed from this repo.

`tabs` had to have the `/dist` in its path removed to avoid breakage, see [`nextjs-scripts@13.2.0` release notes](https://github.com/hashicorp/nextjs-scripts/releases/tag/13.2.0) for details.